core/block_svg.js

```diff
@@ -31,6 +31,7 @@ goog.require('Blockly.ContextMenu');
 goog.require('Blockly.RenderedConnection');
 goog.require('Blockly.Touch');
 goog.require('Blockly.utils');
+goog.require('Blockly.BlockAnimations');
```

Comment:
Move just below `goog.require('Blockly.Block');`
These imports are in alphabetical order.